pipeline/metadata/flatten_satellite.py

```diff
@@ -371,54 +463,60 @@ def _process_satellite_v2_control(
         Rows
     """
     responses = scan.get('response', [])
-    if responses:
-      # An overall satellite v2 measurement
-      # always contains some non-control trial domains
-      is_control_domain = False
+
+    for response in responses:
+      is_control_domain = flatten_base.is_control_url(response['url'])
 
       row = {
           'domain':
-              scan['test_url'],
+              response['url'],
```

Comment:
Extract the domain from the URL.
Why is the field called URL?

Reply:
I'm happy to do either of those name changes. But both of them require a schema change, so they have to happen in https://github.com/censoredplanet/censoredplanet-analysis/pull/119.

Currently, I need to preserve the full content of this field so I'll be able to investigate it and figure out why these are appearing as urls at all. If I do the extraction here (in this PR) then that data will become impossible to access.

Reply:
I don't see how adding a column would be a problem. It actually concerns me that we may not be aggregating correctly if it's urls and not domains.
I'm fine with punting on that for now, but please add a TODO

Reply:
TODO added